source/_docs/scripts/conditions.markdown

```diff
@@ -7,6 +7,7 @@ redirect_from: /getting-started/scripts-conditions/
 Conditions can be used within a script or automation to prevent further execution. When a condition does not return true, the script or automation stops executing. A condition will look at the system at that moment. For example, a condition can test if a switch is currently turned on or off.
 
 Unlike a trigger, which is always `or`, conditions are `and` by default - all conditions have to be true.
+When using lists in conditions, multiple entities default as `and` (only if all entities match) while multiple states default as `or` (if one of the states match).
```

Comment:
This feels out of context here.

This is talking about conditions in general, not about specifics condition types or their features?

Reply:
ok.. For me, it really wasn't clear how it worked until you spelled it out in the tweet so I figured I would try to add that context somewhere in writing.  I chose this spot since in line 9 you talked about the and/or difference between trigger/conditions.   The examples you have laid out in the states section are very clear once you have the right context (at least for me).  That's why I felt like a top level statement would be useful. 

But you can scrap it if you feel as though the docs are sufficient.  I just figured I'd give you the option to add it since I was confused. :)

Reply:
Well you line is about the state condition type specifically, which is described here:

<https://www.home-assistant.io/docs/scripts/conditions/#state-condition>

Reply:
Fair enough.